docs/rfcs/managed.md

```diff
@@ -0,0 +1,142 @@
+# Managed CRD
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
+that covers the feature set of the OpenTelemetry Helm chart.
+
+## Summary
+This request for comment aims to establish a new approach to management of resources in Kubernetes clusters,
+with the explicit goal of taking over the whole cluster, and making decisions
+on how to best instrument all resources in the cluster on behalf of the administrator based on best practices,
+OpenTelemetry standards and semantic conventions.
+
+The operator project offers custom resources (CR) to allow Kubernetes operators to define OpenTelemetry components
```

Comment:
I'd change from Kubernetes operators to Kubernetes users/administrators to avoid confusion with the Kubernetes operators concept.

Reply:
```suggestion
The operator project offers custom resources (CR) to allow Kubernetes users/administrators to define OpenTelemetry components
```